docs/Queries.md

```diff
@@ -90,8 +121,14 @@ A query response is also a plain object
   ref: string,
   value: any,
   type?: string,
-  flags?: array,
-  meta?: object,
+  meta?: {
+    flags?: string[],
+    variants?: {
+      [string]: {  // experiment
+        [string]: string  // context: variant
```

Comment:
actually, supporting a `number` in the parsed variants header would be a little awkward, e.g.

```js
const contextInt = parseInt(context, 10);
if (!isNaN(contextInt)) {
  // use`contextInt`
} else {
  // use `context`
}
```

so I'm inclined to force the _response_ state to be string-keyed, but allow the _request_ to use integer `context`s.